src/gir_typelib.c

```diff
@@ -939,6 +949,79 @@ scm_dump_all_arg_types(void)
 }
 #endif
 
+static void
+scm_typelib_do_define_module (void *data)
+{
+    const char **real_data = data;
+    scm_i_typelib_load (real_data[0], real_data[1]);
+}
+
+static SCM
+scm_typelib_define_module (SCM s_lib, SCM s_version)
+{
+    gchar *name, *data[2];
+
+    SCM_ASSERT_TYPE(scm_is_string(s_namespace), s_namespace, SCM_ARG1, "typelib-load", "string");
+    SCM_ASSERT_TYPE(scm_is_string(s_version), s_version, SCM_ARG2, "typelib-load", "string");
```

Comment:
Did you mean to use "typelib-load" as function name here?

Reply:
No, that was a copypasting error. It should be fixed in 5465cd3.